stable/ibex-dashboard/templates/storageclass.yaml

```diff
@@ -0,0 +1,15 @@
+{{- if and .Values.persistence.enabled (not .Values.persistence.existingClassName) -}}
```

Comment:
A storage class should not be part of the chart. This is something you manage separately on broader scope, not on a per-chart basis.